root/opt/suricata/suricata.yaml

```diff
@@ -177,13 +177,7 @@ magic-file: /usr/share/file/magic
 
 # af-packet support
 # Set threads to > 1 to use PACKET_FANOUT support
-af-packet:
-  - interface: eth0
-    threads: 1
-    cluster-id: 99
-    cluster-type: cluster_cpu
-    defrag: yes
-    use-mmap: yes
+include: af-packet.yaml
```

Comment:
This makes use of Suricata's [split configuration file](http://suricata.readthedocs.io/en/latest/configuration/suricata-yaml.html#splitting-configuration-in-multiple-files) functionality. I first tried using PyYAML to re-write the whole file, but I ran into problems even when reading/re-writing the configuration file with that library.